persister/whisper.go

```diff
@@ -188,7 +213,16 @@ func fnv32(key string) uint32 {
 	}
 	return hash
 }
-
+func (p *Whisper) registerOutOfOrderLags(points []*whisper.TimeSeriesPoint) {
```

Comment:
guess we could keep name similar to prom metric, so `registerOutOfOrderWriteLags`

Reply:
thanks, fixed